Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ruby:2.6.0
```

Comment:
Same version discrepancy, are we on `2.6.0` or `2.5.3`? Perhaps it does not matter.

Reply:
DItto.